### PR TITLE
Adding helper methods for output to stderr and stdout.

### PR DESCRIPTION
**Pull request by levigroker**

### CODE REVIEW

File: CLIKit/CLIApplication.h [hunk 1]
```diff
@@ -140,6 +140,32 @@
  */
 - (NSString*)generateUsageMessage;
 
+/**
+ * Writes the usage message returned by `generateUsageMessage` to stderr.
+ */
+- (void)writeUseageMessage;
```
Comment (umjames) [inline CLIKit/CLIApplication.h:146]: @levigroker Should be renamed to -writeUsageMessageToStandardError;

File: CLIKit/CLIApplication.h [hunk 2]
```diff
@@ -140,6 +140,32 @@
...
+ * @param error The error whose localized description to write to standard error.
+ */
+- (void)writeError:(NSError *)error;
+
+/**
+ * Write a given string to stderr.
+ *
+ * @param string The string to write to standard error.
+ */
+- (void)writeStdErrorString:(NSString *)string;
```
Comment (umjames) [inline CLIKit/CLIApplication.h:160]: @levigroker Should be renamed to - (void)writeToStandardError: (NSString*)string;

File: CLIKit/CLIApplication.h [hunk 3]
```diff
@@ -140,6 +140,32 @@
...
+
+/**
+ * Write a given string to stdout.
+ *
+ * @param string The string to write to standard out.
+ */
+- (void)writeStdOutString:(NSString *)string;
```
Comment (umjames) [inline CLIKit/CLIApplication.h:167]: @levigroker Should be renamed to - (void)writeToStandardOutput: (NSString*)string;

**Comment by umjames** [comment]
@levigroker You will also need to re-issue this pull request against the development branch instead of master.

**Comment by levigroker** [comment]
Updated with suggested changes.

**Comment by levigroker** [comment]
This should be good to go. Please review.

**Pull request closed without merging**